pkg/object/s3.go

```diff
@@ -55,6 +55,9 @@ type s3client struct {
 }
 
 func (s *s3client) String() string {
+	if s.s3.Options().BaseEndpoint != nil {
+		return fmt.Sprintf("%s/%s/", *s.s3.Options().BaseEndpoint, s.bucket)
```

Comment:
Can we use `s3://endpoint/bucket`?

Reply:
`BaseEndpoint` already has the `s3://` prefix or the default `http://` prefix - there is no need to add another one.

Reply:
I also added a UT to demonstrate the output.

Reply:
`BaseEndpoint` always has a schema, even if user does not specify one, the constructor will prepend a default schema:
https://github.com/juicedata/juicefs/blob/f08f36a308ea53e9bd91781209172a58d50a2401/pkg/object/s3.go#L441-L447

Or do you prefer this string to always start with `s3://`? But in that case, it would override the bucket parameter specified by user.

Reply:
yes, we want always use s3:// to keep it consistent.

Reply:
Updated